python-package/lightgbm/sklearn.py

```diff
@@ -621,20 +632,15 @@ def fit(self, X, y,
                     _log_warning(f"Found '{alias}' in params. Will use it instead of 'eval_at' argument")
                     eval_at = params.pop(alias)
             params['eval_at'] = eval_at
-        params['objective'] = self._objective
-        if self._fobj:
-            params['objective'] = 'None'  # objective = nullptr for unknown objective
 
         # Do not modify original args in fit function
         # Refer to https://github.com/microsoft/LightGBM/pull/2619
         eval_metric_list = copy.deepcopy(eval_metric)
         if not isinstance(eval_metric_list, list):
             eval_metric_list = [eval_metric_list]
-
```

Comment:
I don't support this line and the three others below it removing blank lines. I think the resulting block of code is more difficult to read than the current state.

**`master`**

![image](https://user-images.githubusercontent.com/7608904/140001277-172d7433-6f75-4cfe-a62f-5b2e473d858f.png)


**this branch**

![image](https://user-images.githubusercontent.com/7608904/140001186-5e0c5234-e9c0-446b-98c4-c66133769628.png)

Reply:
Ah, OK. Reverted in 771cb5859c704f5e2ed7a58d8abb5a2744604b68.